activerecord/lib/active_record/migration.rb

```diff
@@ -135,6 +135,16 @@ def initialize(message = nil)
     end
   end
 
+  class ConcurrentMigrationError < MigrationError #:nodoc:
+    def initialize(message = nil)
+      if !message
+        super('Cannot run migrations because another migration process is currently running.')
+      else
+        super
+      end
+    end
```

Comment:
Do you think this would be better written as:

```ruby
class ConcurrentMigrationError < MigrationError #:nodoc:
  DEFAULT_MESSAGE = "Cannot run migrations because another migration process is currently running.".freeze

  def initialize(message = DEFAULT_MESSAGE)
    super
  end
end
```